### PR TITLE
#163670520 Fix all endpoint responses

### PR DESCRIPTION
**Pull request by chuxmykel**
#### What does this PR do?
- Remove debug from server file to make running app faster
- Rewrite endpoint responses to match specifications from the ADC
- Change edit route http verb from put to patch to match specification

#### Description of Task to be completed?
- Fix endpoints to work according to Endpoint Specification

#### How should this be manually tested?
- Clone the repo, install using ```npm install``` and run using ```npm start```. Then use postman to target all the routes
#### Any background context you want to provide?
- None
#### Screenshots (if appropriate)
- None

### CODE REVIEW

File: server/app/controllers/partyController.js [hunk 1]
```diff
@@ -9,99 +7,119 @@ class PartyController {
...
-      error: 'Resource not found'
+      error: 'Party does not exist',
     });
   }
 
   editParty(req, res) {
-    const id = parseInt(req.params.id);
-    const { name, hqAddress, logoUrl } = req.body;
+    const id = parseInt(req.params.id, 10);
+    const { name } = req.params;
```
Comment (veeqtor) [inline server/app/controllers/partyController.js]: You are not supposed to get the name from the URL params, you should get it from the JSON body.

File: server/app/routes/index.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const officeEndPoint = '/api/v1/offices/';
 router.post(partyEndPoint, PartyController.addParty);
 router.get(partyEndPoint, PartyController.getAllParties);
 router.get(`${partyEndPoint}:id`, PartyController.getOneParty);
-router.put(`${partyEndPoint}:id`, PartyController.editParty);
+router.patch(`${partyEndPoint}:id/:name`, PartyController.editParty);
```
Comment (veeqtor) [inline server/app/routes/index.js]: The format for the URL is as follows `/<party-id>/name`, please correct this.

Comment (chuxmykel) [reply server/app/routes/index.js]: Done

File: server/app/controllers/partyController.js (hunk 1 above)
Comment (chuxmykel) [reply server/app/controllers/partyController.js]: Done

**Comment by chuxmykel** [comment]
Changes have been implemented

**Comment by veeqtor** [comment]
Looks Good, You can merge.

**Pull request merged**